main.py

```diff
@@ -0,0 +1,7 @@
+##HOW DO YOU USE THE RE.SUB??????????
+
+string=input(("Enter a string:"))
+if(string==string[::-1]):
+      print("The string is a palindrome")
+else:
+      print("Not a palindrome")
```

Comment:
You got some great things going on here Bettina! Now it's time to think about putting it into functions to make it reusable. Great start on this, though!